docker-compose.dev.yml

```diff
@@ -27,6 +27,7 @@ services:
       - WEB_SERVER_HOST=0.0.0.0
       - CRON_MODE=
       - STUDY_CONFIG=stage-program
+      - DYNAMIC_CONFIG=dev-emulator-study
```

Comment:
the STUDY_CONFIG **is** dynamic config. I am not sure why you feel the need to add a second one.

Reply:
Removed DYNAMIC_CONFIG = dev-emulator-study, and replaced STUDY_CONFIG as dev-emulator-study.